frontend/src/pages/GroupPage.jsx

```diff
@@ -117,7 +117,7 @@ export default function GroupPage() {
           }}
           onClick={() => navigateTo(`/${currentNetwork}/groups/create-group`)}
         >
-          Create Group
+          New Group +
```

Comment:
```suggestion
          New Group
```

Use `Button` `startIcon` prop.